app/controlplane/pkg/biz/orgmetrics.go

```diff
@@ -30,12 +30,12 @@ type OrgMetricsUseCase struct {
 
 type OrgMetricsRepo interface {
 	// Total number of runs within the provided time window (from now)
-	RunsTotal(ctx context.Context, orgID uuid.UUID, timeWindow time.Duration) (int32, error)
+	RunsTotal(ctx context.Context, orgID uuid.UUID, timeWindow TimeWindow) (int32, error)
```

Comment:
in general I tend to pass pointers to struct instead of a copy of the struct itself. Should we change that in both the params on the repo methods and the use-case ones?

Reply:
Will do that yes.